registry/testcases/test_variables.py

```diff
@@ -9,7 +9,7 @@
             "apartment": "ABC Apartment",
             "city": "TestCity",
             "state": "state",
-            "pincode": 123456,
+            "pincode": "123456",
```

Comment:
Why pincode is stored as a string?

Reply:
because many countries have pin code as alpha-numeric.